chromium/storage.js

```diff
@@ -1,16 +1,22 @@
-/* exported storage */
-/* global initializeStoredGlobals */
-/* global exports */
+(function(exports) {
+
+Object.assign(exports, {
+  get: chrome.storage.local.get,
+  set: chrome.storage.local.set,
+});
 
-var storage = chrome.storage.local;
 if (chrome.storage.sync) {
   chrome.storage.sync.set({"sync-set-test": true}, () => {
     if(!chrome.runtime.lastError){
-      storage = chrome.storage.sync;
-      initializeStoredGlobals();
+
+      Object.assign(exports, {
+        get: chrome.storage.sync.get,
+        set: chrome.storage.sync.set,
```

Comment:
@Hainish this was the only kind of gross part. In other parts of the code we refer to `storage.get/set`, but they are set in an **asynchronous** way. So I can't just do the exports at the end of the file. Looking back now, I think I should just set them with a promise. I'm going to change this.

* directly set these like `window.storage.get/set = ...get/set`
* or do this, I chose